conf/pulsar_tools_env.sh

```diff
@@ -42,6 +42,18 @@
 # PULSAR_GLOBAL_ZK_CONF=
 
 # Extra options to be passed to the jvm
+# Discard parameter "-Xms" of $PULSAR_MEM, which tends to be the Broker's minimum memory, to avoid using too much
+# memory by tools.
+if [ -n "$PULSAR_MEM" ]; then
+  PULSAR_MEM_REWRITE="-Xmx128m -XX:MaxDirectMemorySize=128m"
```

Comment:
This shell seems to set PULSAR_MEM to `-Xmx128m -XX:MaxDirectMemorySize=128m`?

Reply:
If `PULSAR_MEM` exists, `-Xmx` and `-XX:MaxDirectMemorySize` will be overridden to `128m`, and other options in `PULSAR_MEM` will not be changed.

Reply:
We can just remove the `-Xmx`, `-Xmx` and `-XX:MaxDirectMemorySize` don't need overriding to 128m